contracts/src/v0.8/keystone/CapabilityRegistry.sol

```diff
@@ -64,6 +77,19 @@ contract CapabilityRegistry is OwnerIsCreator, TypeAndVersionInterface {
     }
   }
 
+  /// @notice Removes a node operator
+  /// @param nodeOperatorIds The IDs of the node operators to remove
+  function removeNodeOperators(uint256[] calldata nodeOperatorIds) external {
+    for (uint256 i; i < nodeOperatorIds.length; ++i) {
+      uint256 nodeOperatorId = nodeOperatorIds[i];
+      NodeOperator memory nodeOperator = s_nodeOperators[nodeOperatorId];
+      if (nodeOperator.admin == address(0)) revert NonExistentNodeOperator(nodeOperatorId);
```

Comment:
We could probably skip instead of reverting. Is there harm in not reverting?

Reply:
No harm.  Just removed this check